lib/scss/stanford_events.node.scss

```diff
@@ -0,0 +1,133 @@
+@charset 'UTF-8';
+@import "./stanford_events.config";
+
+.section-editorial-content {
+  @include modular-spacing('padding', 6 null 3 null);
+  @include modular-spacing('margin', null null 5 null);
+
+  color: $su-color-black;
+
+  h1 {
+    @include modular-spacing('margin', null null -1 null);
+    @include modular-typography(5);
+  }
+
+  .su-event-subheadline {
+
+  }
+
+  .su-event-dek {
+    @include big-paragraph;
+    @include modular-spacing('margin', null null 4 null);
+  }
+
+  .su-event-sponsor {
+    @include subheading;
+  }
+}
+
+.body-region {
```

Comment:
This selector doesn't seem specific enough. It is possible that when a view loads an event node as a rendered entity that this stylesheet could be included. Perhaps nest this selector in the layout selector?

Reply:
@jenbreese See above comment.

Reply:
actually, you are not applying styles to the body-region at all here. Is it needed to increase specificity?

Reply:
Yes, because we have the event type on the page twice. I can look at another way too.

Reply:
naw, this is fine for now.